### DISCUSSION TITLE
how to change label of a button by code?

### DISCUSSION THREAD

**Discussion by horstjens** [marimo-team/marimo#3175] [Q&A]
Is it possible to change attributes of an marimo.ui element by code during runtime?
For example, i have an button and want to change it's label or the "disabled" attribute by code.

**Comment by mscolnick** [comment]
You cannot currently do so with our button. We only allow changing these attributes at construction time. However, we do plan to improve this, but not in the near-term.

Instead, you could create your own button or elements using [anywidget](https://anywidget.dev/), which does support updating attributes after being created.
